ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -411,7 +412,8 @@ static KsqlRestApplication buildApplication(
         (ec, sc) -> InjectorChain.of(
             new DefaultSchemaInjector(
                 new SchemaRegistryTopicSchemaSupplier(sc.getSchemaRegistryClient())),
-            new DefaultTopicInjector(ec)
+            new TopicCreateInjector(ec),
```

Comment:
As per comment on `KsqlContextTestUtil`, we can drop `sc` here as its accessible from `ec`.